src/metabase/models/permissions.clj

```diff
@@ -573,7 +573,7 @@
   (set-has-full-permissions? permissions-set (application-perms-path perm-type)))
 
 (s/defn perms-objects-set-for-parent-collection :- #{Path}
-  "Implementation of `IModel` `perms-objects-set` for models with a `collection_id`, such as Card, Dashboard, or Pulse.
+  "Implementation of `perms-objects-set` for models with a `collection_id`, such as Card, Dashboard, or Pulse.
```

Comment:
`perms-objects-set` was never part of `IModel` and especially isn't now since it's a multimethod